README.md

```diff
@@ -88,18 +91,14 @@ $ export LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"      # Linux, Cygwin,
 $ export DYLD_LIBRARY_PATH="/usr/local/lib:$DYLD_LIBRARY_PATH"  # MacOS-X
 ```
 
-#### Uninstall Exiv2 on a UNIX-like system
```

Comment:
I removed the Uninstall section because I think it is not relevant. CMake actually does not actually provide a target to uninstall files.